### PR TITLE
Add note about existing PriorityClasses

### PR DESCRIPTION
**Pull request by bsalamat**
/sig scheduling

### CODE REVIEW

**Comment by bsalamat** [comment]
@tengqm @steveperry-53 

Can you please take a look at this? It would be great to merge this ASAP.

**Comment by steveperry-53** [comment]
@bsalamat Please change the base from release-1.11 to master. Thanks.

File: content/en/docs/concepts/configuration/pod-priority-preemption.md [hunk 1]
```diff
@@ -111,12 +111,27 @@ object for each such mapping that they want.
...
 
 The `description` field is an arbitrary string. It is meant to tell users of
 the cluster when they should use this PriorityClass.
 
+Kubernetes comes with two predefined PriorityClasses: `system-node-critical`, and
+`system-cluster-critical`. The former has the highest priority and is used for critical
+system Pods that must run on all or most of the nodes and should never
+be preempted. The latter is used for system critical Pods that must always
+run in a cluster, but they can be terminated from one node and started on a
+different node.
```
Comment (steveperry-53) [inline content/en/docs/concepts/configuration/pod-priority-preemption.md]: terminated from -> terminated on

File: content/en/docs/concepts/configuration/pod-priority-preemption.md [hunk 2]
```diff
@@ -111,12 +111,27 @@ object for each such mapping that they want.
...
+
+{{< note >}}
+**Note**: You should never use these PriorityClasses that are dedicated to
+system components for non-system Pods. Otherwise, Pods with such high priorities
+may preempt system critical components causing disruption of cluster services.
```
Comment (steveperry-53) [inline content/en/docs/concepts/configuration/pod-priority-preemption.md]: may -> might
components causing -> components, causing

**Comment by steveperry-53** [comment]
I put a few comments in the text.
/approve for writing.
Still need `/lgtm` for technical approval.

File: content/en/docs/concepts/configuration/pod-priority-preemption.md [hunk 3]
```diff
@@ -111,12 +111,27 @@ object for each such mapping that they want.
 PriorityClass also has two optional fields: `globalDefault` and `description`.
 The `globalDefault` field indicates that the value of this PriorityClass should
 be used for Pods without a `priorityClassName`. Only one PriorityClass with
-`globalDefault`  set to true can exist in the system. If there is no PriorityClass
+`globalDefault` set to true should exist in the system. If there are more than one
+PriorityClasses with `globalDefault` set to true, the value of the smallest
```
[the remark this answers is not in the document]
Comment (bsalamat) [reply content/en/docs/concepts/configuration/pod-priority-preemption.md]: done

File: content/en/docs/concepts/configuration/pod-priority-preemption.md [hunk 4]
```diff
@@ -111,12 +111,27 @@ object for each such mapping that they want.
...
+of such PriorityClasses is used as the default priority. If there is no PriorityClass
```
[the remark this answers is not in the document]
Comment (bsalamat) [reply content/en/docs/concepts/configuration/pod-priority-preemption.md]: done

File: content/en/docs/concepts/configuration/pod-priority-preemption.md (hunk 1 above)
Comment (bsalamat) [reply content/en/docs/concepts/configuration/pod-priority-preemption.md]: done

File: content/en/docs/concepts/configuration/pod-priority-preemption.md (hunk 2 above)
Comment (bsalamat) [reply content/en/docs/concepts/configuration/pod-priority-preemption.md]: done

**Comment by bsalamat** [comment]
@k82cn Could you please review the changes here?

File: content/en/docs/concepts/configuration/pod-priority-preemption.md [hunk 5]
```diff
@@ -111,12 +111,27 @@ object for each such mapping that they want.
...
+PriorityClass with `globalDefault` set to true, the value of the smallest
+of those PriorityClasses is used as the default priority. If there is no PriorityClass
 with `globalDefault` set, the priority of Pods with no `priorityClassName` is zero.
 
 The `description` field is an arbitrary string. It is meant to tell users of
 the cluster when they should use this PriorityClass.
 
+Kubernetes comes with two predefined PriorityClasses: `system-node-critical`, and
+`system-cluster-critical`. The former has the highest priority and is used for critical
+system Pods that must run on all or most of the nodes and should never
```
Comment (k82cn) [inline content/en/docs/concepts/configuration/pod-priority-preemption.md]: nits: it'd like to say " system-node-critical ... critical system Pods that much run specific node and should never ...", as priority is bound to pod; "all or most of nodes" seems related to DaemonSet or Deployment, although that maybe the case :)

File: content/en/docs/concepts/configuration/pod-priority-preemption.md [hunk 6]
```diff
@@ -111,12 +111,27 @@ object for each such mapping that they want.
...
+be preempted. The latter is used for system critical Pods that must always
+run in a cluster, but they can be terminated on one node and started on a
+different node.
+
+{{< note >}}
+**Note**: You should never use these PriorityClasses that are dedicated to
+system components for non-system Pods. Otherwise, Pods with such high priorities
```
Comment (k82cn) [inline content/en/docs/concepts/configuration/pod-priority-preemption.md]: Do we need to highlight that, the admission controller will help on that?

**Comment by k82cn** [comment]
oh, @steveperry-53 , I'd like to say it's LGTM overall; those comments are nice to have, according to schedule :)

/lgtm

**Pull request merged**